releasenotes/notes/quadratic_model_scale_method-52f5a73ec045801c.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add new `scale` method to `QuadraticModelBase`. Eventually to be ported to python
```

Comment:
I think this can be more simply
```
Add `QuadraticModelBase::scale` method. Scales the offset, linear biases and quadratic biases by a provided scale factor.
```
chances are the port will be included in the same release so I don't think we need to say anything about it.

Reply:
Thanks